clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/BaseGVFSOperations.java

```diff
@@ -537,6 +553,13 @@ private void setCallerContext(FilesetDataOperation operation) {
     CallerContext.CallerContextHolder.set(callerContext);
   }
 
+  private void setCallerContextForGetCredentials(String locationName) {
+    Map<String, String> contextMap = Maps.newHashMap();
+    contextMap.put(CredentialConstants.HTTP_HEADER_CURRENT_LOCATION_NAME, locationName);
+    CallerContext callerContext = CallerContext.builder().withContext(contextMap).build();
```

Comment:
What if the `callContext` already existed? Will you try to override it?

Reply:
yes, it will be overrided by current contextMap

Reply:
I means what if the `callContext` is not empty and contains some other properties here, you will remove them all when you add `LOCATION_NAME`.

Reply:
Yes, doing so has no side effects, no other place will use it